dm/ui/src/components/BatchImportTaskConfig/index.tsx

```diff
@@ -0,0 +1,250 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { clamp } from 'lodash'
+import { useSafeState } from 'ahooks'
+import QueueAnim from 'rc-queue-anim'
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter'
+import { ghcolors } from 'react-syntax-highlighter/dist/esm/styles/prism'
+
+import { Button, Radio, Checkbox, Steps, Alert, StepProps } from '~/uikit'
+import {
+  BatchImportTaskConfigResponse,
+  useDmapiBatchImportTaskConfigMutation,
+} from '~/models/taskConfig'
+import { sleep } from '~/utils/sleep'
+
+const { Step } = Steps
+
+const BlinkingText: React.FC<{ text: string; active: boolean }> = ({
+  text,
+  active,
+}) => {
+  const [count, setCount] = useSafeState(1)
+  let timer = useRef<number>()
+
+  useEffect(() => {
+    if (active) {
+      timer.current = window.setInterval(() => setCount(c => c + 1), 300)
```

Comment:
Maybe better to `setCount( c => (c+1)%4)` to avoid nubmer value overflow? even very very low possibility.